src/vs/workbench/contrib/search/browser/patternInputWidget.ts

```diff
@@ -154,15 +170,19 @@ export class PatternInputWidget extends Widget {
 	private onInputKeyUp(keyboardEvent: IKeyboardEvent) {
 		switch (keyboardEvent.keyCode) {
 			case KeyCode.Enter:
-				this._onSubmit.fire(false);
+				this._onSubmit.fire();
 				return;
 			case KeyCode.Escape:
-				this._onCancel.fire(false);
+				this._onCancel.fire();
 				return;
 			default:
 				return;
 		}
 	}
+
+	get searchConfig() {
```

Comment:
Seems random to have this as a public property on patternInputWidget, maybe make it private

Reply:
Good point, I will fix that.

Reply:
Fixed.